backend/src/resolvers.ts

```diff
@@ -1,25 +1,70 @@
-export const resolvers = {
-  // for Post and User we can -maybe- use default resolvers
+import { delegateToSchema } from 'graphql-tools'
+
+export default ({ subschema }) => ({
   Query: {
-    posts: (parent, { newsId }, context, info) => {
-      return context.dataSources.posts.getPosts()
+    posts: async (parent, args, context, info) => {
+      const post = await delegateToSchema({
+        schema: subschema,
+        operation: 'query',
+        fieldName: 'Post',
+        context,
+        info
+      })
+      return post
     },
-    users: (parent, { userId }, context, info) => {
-      return context.dataSources.posts.getUsers()
+    users: async (parent, args, context, info) => {
+      const user = await delegateToSchema({
+        schema: subschema,
+        operation: 'query',
+        fieldName: 'User',
+        context,
+        info
+      })
+      return user
     }
   },
   Mutation: {
```

Comment:
⭐ ⭐ Data created in mutations are persisted and your queries and mutations still work as expected.